ImageEncoding/src/ImageEncoding.jl

```diff
@@ -0,0 +1,27 @@
+module ImageEncoding
+
+using ImageBase: restrict
+using ImageCore
+using Requires
+using Crayons
+
+export
+  TermColorDepth, TermColor256, TermColor24bit,
+  ImageEncoder, BigBlocks, SmallBlocks,
+  colorant2ansi, encodeimg, sixel_encode
```

Comment:
I hope we can make this package simpler, absorb the "whether big-or-small Blocks" part in `imshow`, and only export `ascii_encode`.

We should also provide a tool `is_24bit_supported` to simplify the `ImageInTerminal.__init__()`.

Reply:
I moved the `BigBlocks` `SmallBlocks` logic to `ascii_encode`.